### DISCUSSION TITLE
It's not working

### DISCUSSION THREAD

**Discussion by Markospoko** [Qalculate/libqalculate#723] [General]
Hi,
I installed latest version and Qalculate classic doesn't start, nothing shows.
I've Windows 10 Pro.

**Comment by hanna-kn** [comment]
Please try removing the program and then install again. It seems as if for some reason the installer does not replace some of the old files (primarily libgdk_pixbuf-2.0-0.dll).

The installer has now been updated.

**Reply by Markospoko** [reply]
OK, that works.
Although I don't know if it's normal that it starts a bit slower.
Thanks

**Reply by hanna-kn** [reply]
Are you sure that it consistently starts slower? There are no changes in Qalculate which should reasonably make it slower.

**Reply by Markospoko** [reply]
I'm unable to compare with before - Qalculate Qt opens immediately but it takes Qalculate classic 2s or something like that to open.

**Reply by hanna-kn** [reply]
It is normal that the Gtk (classic) version is slower than the Qt version.

**Reply by Markospoko** [reply]
That's what I thought. But perhaps you are measuring which clothes would have close size in these few seconds... - only joking.
